README.md

```diff
@@ -44,29 +44,29 @@ This module can be pulled from Maven Central:
 ```
 
 ## Building
-This is a maven project which you can build using
+This is a Maven project which you can build using the following command:
 ```bash
 mvn clean install
 ```
 
 ## Environment
 
-To use these providers you need to have the Python 3.6 with the following packages installed on your environment:
+To use these providers you need to have Python 3.6 with the following packages installed in your environment:
```

Comment:
I think we need an empty line before the bullets, otherwise it doesn't render the markdown properly (see https://github.com/krisztinaknagy/feedzai-openml-python/blob/66529ee58b656be8687ab14952ee2b3d652c29f8/README.md for a preview)